wof/core.py

```diff
@@ -27,6 +21,18 @@
 from spyne.server.http import HttpTransportContext
 from spyne.server.wsgi import WsgiApplication
 
+import sys
+print('sys.path!!!!')
+print(sys.path)
```

Comment:
I'm guessing this is left over debugging statements.

PS: using env will give you path isolation and reduce headaches.

Reply:
Yup, thanks, removed now.